overlay.py

```diff
@@ -187,8 +188,22 @@ def on_connect(self, client, userdata, flags, rc):
         """ Called automatically when the overlay connects successfully to the
             MQTT broker.
 
-            Overlay implementations may override for one-off operations
-            (e.g. drawing self.base_canvas) """
+            Overlay implementations may override for one-off operations."""
+        pass
+
+    def display_base_layer(self):
+        """ Set up the base layer as soon as camera turns on.
+
+            Method only needs to be called once. Should also catch any errors
+            that occur when base layer is displayed. """
+        with self.exception_handler:
+            self._display_base_layer()
+
+    def _display_base_layer(self):
```

Comment:
This can be abstract! Also, we shouldn't need `pass`, as the docstring is sufficient method content. (there's a few other extra `pass`es in this file, remove them too if you feel like it haha)